Directory.Build.props

```diff
@@ -85,6 +85,7 @@
   <PropertyGroup>
     <RestoreSources Condition="'$(DotNetBuildOffline)' != 'true'">
       https://dotnetfeed.blob.core.windows.net/dotnet-core/index.json;
+      https://dotnetfeed.blob.core.windows.net/dotnet-tools-internal/index.json;
```

Comment:
We should put this package on an existing feed instead of adding this feed to the list.